addons/web_editor/static/src/scss/web_editor.variables.scss

```diff
@@ -36,15 +36,18 @@ $o-we-ace-color: #2F3129 !default;
 // Needed to be changed to be high enough to not overflow when a user
 // has a page with a lot of content (10000px was proven to be too small)
 $o-we-handles-offset-to-hide: 100000px !default;
-$o-we-handles-btn-size: 20px !default;
+$o-we-handles-btn-size: 14px !default;
```

Comment:
I'm not sure these elements should be the same size. The `oe_overlay` elements and the sidebar are two separate parts of the UI.
![Capture d’écran 2023-09-07 à 13 46 46](https://github.com/odoo/odoo/assets/80679690/4b0f2781-6918-4565-9db3-d65c875abc82)

In addition, the calculation to create the inner line must have an even value, otherwise rendering is not good.
![Capture d’écran 2023-09-07 à 13 53 37](https://github.com/odoo/odoo/assets/80679690/b59b6085-c11d-4f20-bb52-ebbcb5a037b9)